_posts/en/newsletters/2021-02-17-newsletter.md

```diff
@@ -0,0 +1,307 @@
+  goal is to allow both a hardware wallet and the computer controlling
+  it to each be assured that the nonce used to generate a signature is
+  actually an unguessable value.  This prevents malicious hardware
+  wallet firmware from generating nonces known to the firmware author,
+  which the author could combine with your signatures found onchain to
+  derive your private keys, allowing them to spend any other bitcoins
+  controlled by those keys.  The post describes the technique used,
+  which was previously mentioned in a mailing list thread about this
+  subject almost a year ago (see Newsletters [#87][news87 exfil] and
+  [#88][news88 exfil]).
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Blockstream announces LNsync:**
+  [LNsync][blockstream blog lnsync] allows Lightning wallets to more quickly sync the
+  LN topology in order to optimally route a payment to its destination. An open
+  source plugin, [historian][github historian], provides this functionality for
+  C-Lightning users.
+
+- **Rust light client Nakamoto released:**
+  Alexis Sellier [released Nakamoto][nakamoto blog], a "high-assurance Bitcoin
+  light-client implementation in Rust, with a focus on low resource utilization,
+  modularity and security".
+
+- **Blockstream Satellite broadcasting LN data, Bitcoin Core source:**
+  Blockstream's Satellite, in addition to broadcasting Bitcoin blockchain data,
+  now includes the [Bitcoin Core source code][blockstream satellite bitcoin
+  code], code for a satellite-optimized Bitcoin fork, [Bitcoin
+  Satellite][github bitcoin satellite], and [LN gossip data][blockstream
+  satellite ln data].
+
+- **Blockstream Green implements CSV:**
+  Green Wallet has previously used nLockTime as a wallet recovery mechanism,
+  requiring users to receive backup emails with pre-signed transactions from
+  Blockstream after each transaction in order to recover funds. By [implementing
+  an `OP_CHECKSEQUENCEVERIFY` (CSV) recovery mechanism][blockstream green csv],
+  wallets can be recovered without needing backup files or associating an email
+  address with the wallet.
+
+- **Muun 2.0 released:**
+  The latest [Muun 2.0 release][muun 2.0 blog] includes multisig, wallet recovery
+  features, and also open sources the Android and iOS mobile apps.
+
+- **Joinmarket 0.8.1 released:**
+  The [0.8.1 release][joinmarket 0.8.1] contains additional [PSBT][topic psbt]
+  support for PSBTs created externally, [signet][topic signet] support, a fix
+  for BIP21 (see [Newsletter #127][news127 bech32 casing]). Terminal-based UI
+  for JoinMarket, [JoininBox][github joininbox], was also updated to support 0.8.1.
+
+- **VBTC adds batched segwit withdrawals:**
+  VBTC, a Vietnamese exchange, [added batched segwit withdrawal][vbtc blog
+  segwit] capabilities after recently also [enabling Lightning withdrawals][vbtc blog lightning].
```

Comment:
At first glance, the "Lightning Withdrawals" seem more noteworthy, so maybe that should be the title. Their process for batched withdrawals seems pretty involved, it's not clear to me what message we want to convey by highlighting it.

They charge a flat 1 mBTC fee for withdrawals and now offer this "Batched SegWit" withdrawal option. Withdrawing to a native segwit address gets an 80% discount, making the fee 0.2 mBTC. Other addresses "only" get discounted to 0.05 mBTC (almost certain that was supposed to be 0.5 mBTC). And the batched withdrawal is processed manually by the developers on the following weekend. The most interesting thing to me is that they actually offer financial encouragement to use the most efficient output format, but this is not highlighted in the brief summary.

I think this item should either be dropped or we should try to clarify what we think is noteworthy.

Reply:
I think both the Lightning post (which is from a couple months ago which is why I had it take a back seat here a bit), and the segwit batching are newsworthy. There are some details around their batching process (as all batching has) and how they pass those savings on to their customers but I think adoption of batching and segwit (encouragement) is the main highlight.

I added some different wording here and a bit more detail. Hopefully that helps.

Reply:
Looks much better, thanks!